datasets/streaming/error_aggregates/intro.md

```diff
@@ -1,20 +1,27 @@
-The `error_aggregates` table represents counts of errors counted from main and crash
+The `error_aggregates_v2` table represents counts of errors counted from main and crash
 pings, aggregated every 5 minutes. It is the dataset backing the main [mission
 control](https://data-missioncontrol.dev.mozaws.net/) view, but may also be queried
 independently.
 
 #### Contents
 
-The `error_aggregates` table contains counts of various error measures (for
+The `error_aggregates_v2` table contains counts of various error measures (for
 example: crashes, "the slow script dialog showing"), aggregated across each
 unique set of dimensions (for example: channel, operating system) every 5
 minutes. You can get an aggregated count for any particular set of dimensions
 by summing using SQL.
 
+##### Experiment unpacking
+
+It's important to note that when this dataset is written, pings are "exploded" on (`experiment_id`, `branch_id`) and
+dimension with `null` (`experiment_id`, `branch_id`) is generated for each ping participating in any experiment.
+Therefore care must be taken when writing aggregating queries over the whole population - in these cases one needs to
+filter for `experiment_id=null and branch_id=null` in order to not double-count pings from experiments.
```

Comment:
This paragraph confused me until I looked at your example below. I think this wording might be a little more clear:

"It's important to note that when this dataset is written, pings from clients participating in an experiment are aggregated on the `experiment_id` and `branch_id` dimensions corresponding to what experiment and branch they are participating. However, they are also aggregated with the rest of the population where the values of these dimensions are null. Therefore care must be taken when writing aggregating queries over the whole population - in these cases one needs to filter for `experiment_id=null and branch_id=null` in order to not double-count pings from experiments."

Reply:
Thank you, it is more clear indeed.